cmd/cluster/create_cluster.go

```diff
@@ -165,5 +180,6 @@ func init() {
 	createClusterCmd.Flags().String("cluster-tier", "", "[OPTIONAL] The tier of the cluster. Sandbox or Dedicated. Default Sandbox.")
 	createClusterCmd.Flags().String("fault-tolerance", "", "[OPTIONAL] The fault tolerance of the cluster. The possible values are NONE, ZONE and REGION. Default NONE.")
 	createClusterCmd.Flags().String("database-version", "", "[OPTIONAL] The database version of the cluster. Stable or Preview. Default depends on cluster tier, Sandbox is Preview, Dedicated is Stable.")
+	createClusterCmd.Flags().String("encryption-spec", "", "[OPTIONAL] The customer managed key spec for the cluster. Please provide key value pairs provider=AWS,aws-secret-key=<secret-key>,aws-access-key=<access-key>,aws-arn=<arn1>,aws-arn=<arn2> . If specified, all parameters for that provider are mandatory.")
 
```

Comment:
Is this only for AWS? Can we make it generic like provider=<cloud-provider> and put default value as AWS.

Reply:
Also, maybe make this `cloud-provider`?

Reply:
I will make it `cloud-provider`. It is only AWS for now, GCP coming soon.

Reply:
Also, maybe make it `access-key`, `arn` and `secret-key` and remove the `aws` prefix to make it generic?